images/python/bootstrap.py

```diff
@@ -234,12 +229,17 @@ def filter(self, record):
     stopWithError("Function name is not defined")
 
 try:
-    with file(payloadFileName) as f:
+    if payloadFileName:
+        stopWithError("No payload present")
```

Comment:
file not existing will be captured by the exception. Modifying script to do that. Thanks for catching these errors. Tested all error combinations now (env undefined, env empty, env not a real file, env is bad json, stdin is bad json)